added coupon page (#52)
argoapp-live/argo-docs: versioned_docs/version-3.0.0-beta/billing/coupon.md

---
id: coupon
title: Coupon Activation
sidebar_label: Coupon
slug: /billing/coupon
---

Spheron automatically provides all users with a free Starter package to access and use the platform when you sign up until the user upgrades to the Pro plan.

> To check your spheron Plan [**please click here**](https://docs.spheron.network/billing/plans)

Reviewer: ```suggestion
To check your Spheron Plan [**please click here**](https://docs.spheron.network/billing/plans)
```

> ## How to upgrade your plan using the Coupon code options

Reviewer: Can you add another section for activating coupons from the coupon page by clicking add coupon?

Author: @rekpero  please explain this further.

Reviewer: You can activate coupon by following these step as well:
- Go to billing page, then click coupon tab
- Then click on **Add new coupon** button
- You will get a popup to enter your coupon code, once you fill the coupon code, click on Add button.

I would like you to mention these steps with screenshot in this coupon page in doc

### Step 1: Navigate to the billing page of your Application
<img src={require('@site/static/img/3.0.0/billing-page.png').default} /> <br/><br/>

### Step 2: Click on upgrade plan button
<img src={require('@site/static/img/3.0.0/plan.png').default} /> <br/><br/>

> #### Clicking on the upgrade plan button takes you to the plan options as shown above. Click the switch to pro button

Reviewer: ```suggestion
#### Clicking on the upgrade plan button takes you to the plan options as shown above. Click the Switch to Pro button.
```

### Step 4: Input your coupon code in the Coupon box Option
<img src={require('@site/static/img/3.0.0/coupon-option.png').default} /> <br/><br/>

 **Click on Upgrade plan and wait for the page to load**🔥🔥🔥🔥🔥

## How to know the status of your Coupon code.

> #### On your billing page,click on the coupon button on the menu bar at the left

Reviewer: ```suggestion
#### On your billing page, click on the coupon button on the menu bar at the left
```

<img src={require('@site/static/img/3.0.0/check-coupon.png').default} /> <br/><br/>

:::info

> Users can check the activation and expiry date of their coupon code as well as see active,pending and expired Coupon.

Reviewer: ```suggestion
Users can check the activation and expiry date of their coupon code as well as see active, pending, and expired Coupons.
```

---
id: coupon
title: Coupon Activation
sidebar_label: Coupon
slug: /billing/coupon
---

> Spheron automatically provides all users with a free Starter package to access and use the platform when you sign up until the user upgrades to the Pro plan.

Reviewer: ```suggestion
Spheron automatically provides all users a free Starter package to access and use the platform when you sign up until the user upgrades to the Pro plan. Depending on the coupon's use case, you can also use coupon codes to upgrade your plan or buy bonuses. We would show you how to activate a coupon code if you got one.
```

To check your Spheron Plan [**please click here**](https://docs.spheron.network/billing/plans)

## How to upgrade your plan using the Coupon code options

### Step 1: Navigate to the billing page of your Application
<img src={require('@site/static/img/3.0.0/billing-page.png').default} /> <br/><br/>

### Step 2: Click on upgrade plan button
<img src={require('@site/static/img/3.0.0/plan.png').default} /> <br/><br/>

#### Clicking on the upgrade plan button takes you to the plan options as shown above. Click the Switch to Pro button.

### Step 4: Input your coupon code in the Coupon box Option
<img src={require('@site/static/img/3.0.0/coupon-option.png').default} /> <br/><br/>

 **Click on Upgrade plan and wait for the page to load**🔥🔥🔥🔥🔥

 -------------------------

## How to activate your Coupon code.

> ### Step 1: Go to the billing page of your application, click the coupon tab.

Reviewer: ```suggestion
### Step 1: Go to the billing page of your application, and click the coupon tab.
```

#### Click on add new coupon button as shown below.
<img src={require('@site/static/img/3.0.0/check-coupon.png').default} /> <br/><br/>

> ### Step 2: Add coupon code details as shown in the pop up menu below and click on Add.

Reviewer: ```suggestion
### Step 2: Add coupon code details as shown in the popup menu below and click on Add.
```

<img src={require('@site/static/img/3.0.0/add-coupon.png').default} /> <br/><br/>

> ### Step 3: Wait for the page to reload and viola!!!🔥 see status of your coupon page.

Reviewer: ```suggestion
### Step 3: Wait for the page to reload, and viola!!!🔥 See the status of your coupon page.
```

------

## How to know the status of your Coupon code.

> #### On your billing page, click on the coupon button on the menu bar at the left

Reviewer: ```suggestion
#### On your billing page, click on the coupon tab on the menu bar at the left
```
